gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/JobEvent.java

```diff
@@ -27,7 +27,7 @@
  */
 public class JobEvent {
 
-  public static final String JOB_STATE = "JobStateEvent";
+  public static final String JOB_STATE = "JobStateEvent"; // TODO: Migrate to JobStateEventBuilder
```

Comment:
Since `JobStateEventBuilder ` is newly created, why not use that now ?

Reply:
It is used to emit MRJob status in this pr.  Migrating gobblin job status to use it is a separate task.